__tests__/app.test.js

```diff
@@ -140,3 +138,46 @@ describe("Get /api/articles", () => {
         })
     })
 })
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("200: Should respond with an array of comments for the given article_id with the relevant properties, ordered by most recent comment first", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
```

Comment:
The idea of this that was mentioned in your previous feedback was to check the length of the array we get as a response from the database, rather than the length of the values inside each object.